qutebrowser/browser/downloads.py

```diff
@@ -286,6 +285,13 @@ def _ask_overwrite_question(self):
                                     window=self._win_id)
         message_bridge.ask(q, blocking=False)
 
+    def _download_dir(self):
+        """Get the download directory to use."""
+        download_dir = config.get('storage', 'download-directory')
+        if download_dir is None:
+            download_dir = standarddir.get(QStandardPaths.DownloadLocation)
+        return download_dir + os.sep
```

Comment:
I'm not sure how exactly `os.path.join` will behave on all platforms when one path ends with `os.sep` - so I'd prefer to add that in `path_suggestion` when needed and not here.